### DISCUSSION TITLE
Control characters in output of st2 run

### DISCUSSION THREAD

**Discussion by ahoffmannkvb** [StackStorm/st2#6018] [Q&A]
Hi, 

I am trying to parse the output of "st2 run foo.bar" with yq (a YAML cli tool), but get errors because of the control characters: 

`Error: bad file '-': yaml: control characters are not allowed`

There doesn't seem to be a "--no-color" command line switch. Or am I missing something?

**Comment by nzlosh** [comment]
The default output from `st2` cli is not YAML.  There is an `st2` flag to output the result in yaml that you should use.

**Comment by ahoffmannkvb** [comment]
Ah, yes. Thank you. With "st2 run --yaml foo.bar" it works much better.
